nixos/modules/profiles/minimal.nix

```diff
@@ -21,6 +21,8 @@ with lib;
 
   environment.stub-ld.enable = false;
 
+  fonts.fontconfig.enable = false;
```

Comment:
Maybe it’s best to just disable installation of default fonts, as the configuration file itself takes up little space?

Reply:
I don't know how to do this. Looking at the module there seems to be no way to remove the fonts https://github.com/NixOS/nixpkgs/blob/nixos-unstable/nixos/modules/config/fonts/fontconfig.nix#L287-L357

Reply:
It’s `fonts.enableDefaultPackages`; it’s off by default, and set by e.g. `services.graphical-desktop.enable`. I think we could probably just drop this?